extra_scripts/create_imagenet_r_data_files.py

```diff
@@ -0,0 +1,276 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import argparse
+import importlib
+import os
+
+from torchvision.datasets.utils import download_and_extract_archive
+
+
+def get_argument_parser():
+    """
+    List of arguments supported by the script
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-i",
+        "--input",
+        type=str,
+        help="Path to the folder containing the original imagenet-r folder",
+    )
+    parser.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        help="Folder where the classification dataset will be written",
+    )
+    parser.add_argument(
+        "-d",
+        "--download",
+        action="store_const",
+        const=True,
+        default=False,
+        help="To download the original in the input folder",
+    )
+    return parser
+
+
+def download_dataset(root: str):
+    """
+    Download the Imagenet-R dataset archive and expand it in the folder
+    provided as parameter
+    """
+    URL = "https://people.eecs.berkeley.edu/~hendrycks/imagenet-r.tar"
+    download_and_extract_archive(url=URL, download_root=root)
+
+
+def import_imagenet_a_module():
+    script_path = os.path.split(__file__)[0]
+    return importlib.import_module("create_imagenet_a_data_files", script_path)
+
+
+IMAGENET_R_TARGET_MAPPINGS = {
```

Comment:
Is there a way for us to upload this to AWS and retrieve the mapping from URL
```
module load  fairusers_aws

fs3cmd put my_file.txt s3://dl.fbaipublicfiles.com/vissl/data/my_file.txt
```

Reply:
Yes, that's actually something I wanted to ask you: how to handle properly the resource files 👍 

I like the idea of moving that to AWS ! Thank you Priya :)